ios/brave-ios/Sources/BraveWallet/WalletConstants.swift

```diff
@@ -100,7 +100,11 @@ public struct WalletConstants {
   /// The supported Solana Name Service (SNS) extensions
   static let supportedSNSExtensions = [".sol"]
   /// The supported Unstoppable Domain (UD) extensions
-  public static let supportedUDExtensions = [".crypto", ".x", ".nft", ".dao", ".wallet", ".blockchain", ".bitcoin", ".zil"]
+  public static let supportedUDExtensions = [
+    ".crypto", ".x", ".nft", ".dao", ".wallet",
+    ".blockchain", ".bitcoin", ".zil", ".altimist", ".anime",
+    ".klever", ".manga", ".polygon", ".unstoppable", ".pudgy"
+  ]
```

Comment:
I would say we need that list sorted and in one place. 
We should have a mojo method to backend `ResolveWalletAddress(string address, BlockhainToken token)` which is triggered when address field has a dot.
Not now of course :)

Reply:
++ for both

A few spots we need to expose constants better for all platforms, like currency codes with [#22481](https://github.com/brave/brave-browser/issues/22481) :)

Reply:
1. https://github.com/brave/brave-browser/issues/36165
2. https://github.com/brave/brave-browser/issues/26655